deerlab/fitmultimodel.py

```diff
@@ -208,12 +219,13 @@ def Kmodel(Kpar):
     if len(lbK) is not nKparam or len(ubK) is not nKparam:
         raise ValueError('The upper/lower bounds of the kernel parameters must be ',nKparam,'-element arrays')
 
-    areCenterDistances = [str in ['Mean','Location'] for str in paramnames]
-    if any(areCenterDistances):
+    areLocations = [str in ['Mean','Location'] for str in paramnames]
+    areSpreads   = [str in ['Spread','Width','Standard deviation'] for str in paramnames]

```

Comment:
What is the reason you apply these initialization strategies to the spread? I would think it is needed only for the location. Otherwise, what would you do about other basis function parameters such as asymmetry parameters?

Second question: do all dd model functions have a single location parameter, or there any that have two? This would have implications here.